### PR TITLE
refactor: update filter logic for new projects

### PR DESCRIPTION
**Pull request by jashlu**

### CODE REVIEW

**Comment by jashlu** [comment]
New logic for filtering new projects.

We have two possible fields dedicated to determining period interval to check for new projects. **Days** and **Months**

Days is for schedules like weekly (7 days) or bi-monthly (14 days). Months is for monthly updates. Month is needed because we cant quantify months using days since not every month is the same amount of days

File: src/lib.js [hunk 1]
```diff
@@ -194,9 +200,14 @@ export function filterStale(projects) {
   return filtered
 }
 
-export function filterNew(projects, numberOfDaysBack){
+export function filterNew(projects){
   // projects is an array of objects; numberOfDaysBack is the number of days to search back from day of email
```
Comment (hetd54) [inline src/lib.js:204]: let's edit this comment, since 'numberOfDaysBack' is removed

**Review by hetd54 — COMMENTED** [review]
see my note about changing the comment explaining the newProject function -- otherwise this looks great :) let me know once the comment is changed/removed and I'll approve

File: src/lib.js (hunk 1 above)
Comment (jashlu) [reply src/lib.js:204]: Oh good catch, thanks

**Pull request merged**